src/bindings/java/CMakeLists.txt

```diff
@@ -0,0 +1,161 @@
+find_package(Java REQUIRED)
+find_package(JNI REQUIRED)
+
+set(SOURCES
```

Comment:
What not using a 'glob' to find all the sources?
On long-term, hard-coding the file names introduces unnecessary extra maintenance cost.

Reply:
Globbing in build systems is evil for a number of reasons. This stackoverflow post lists some:

https://stackoverflow.com/questions/1027247/is-it-better-to-specify-source-files-with-glob-or-each-file-individually-in-cmak/1060061#1060061

Another big one is that it ruins the ability to create reproducible builds:
https://reproducible-builds.org/docs/stable-inputs/

Reply:
Really interesting discussion.